db/migrate/20150227185631_add_users_state_flag.rb

```diff
@@ -0,0 +1,15 @@
+class AddUsersStateFlag < ActiveRecord::Migration
+  def up
+    add_column :users, :state, :string, default: 'temp'
+    execute "update users set state='temp' where is_temp='t'"
```

Comment:
Should be fine. Postgresql accepts 't' as a true boolean value.
However, the issue here is that ALL users will be marked as temp (because temp is the default).
I think what we really want is update users set state='activated' where is_temp='f'